documentation/src/main/asciidoc/topics/con_in_memory_caching.adoc

```diff
@@ -0,0 +1,21 @@
+[id='in-memory-caching_{context}']
+= In-memory caching
+
+{brandname} provides applications with data access that has much lower latency for read operations and higher throughput for write operations in comparison with traditional database systems.
+For in-memory caches, {brandname} stores key/value pairs in RAM only.
+
+.In-memory caches
+//Community content
+ifdef::community[]
+image::in-memory-cache-ispn.png[This illustration depicts a {brandname} cluster of four server nodes with in-memory caches that allow applications to perform read and write operations through remote clients.]
+endif::community[]
+//Downstream content
+ifdef::downstream[]
+image::in-memory-cache.png[This illustration depicts a {brandname} cluster of four server nodes with in-memory caches that allow applications to perform read and write operations through remote clients.]
+endif::downstream[]
+
+For in-memory caching deployments, {brandname}:
+
+* Maintains a single copy of each entry for higher performance read operations with lower durability guarantees.
+* Uses eviction or overflow to disk to avoid out of memory exceptions.
+* Clients use near-caching to avoid repeatable reads and gain performance.
```

Comment:
`repeatable reads` <- can we avoid this combination of words? It may confuse users since it is a transactional consistency level :)

Reply:
Fixed